src/LinuxTracing/PerfEvent.h

```diff
@@ -249,6 +249,24 @@ struct DmaFenceSignaledPerfEventData {
 };
 using DmaFenceSignaledPerfEvent = TypedPerfEvent<DmaFenceSignaledPerfEventData>;
 
+struct PerfRecordSample {
+  perf_event_header header;
+  perf_event_sample_id_tid_time_streamid_cpu sample_id;
+
+  mutable uint64_t ips_size;
+  mutable std::unique_ptr<uint64_t[]> ips;
+
+  uint32_t raw_size;
+  std::unique_ptr<uint8_t[]> raw_data;
+
+  uint64_t abi;
+  std::unique_ptr<perf_event_sample_regs_user_all> regs;
```

Comment:
Also `perf_event_sample_regs_user_all` already contains the `abi`.

Reply:
Noted, but this isn't relevant anymore after the new changes.